api/view/static/openapi.json

```diff
@@ -0,0 +1,298 @@
+                }
+              }
+            }
+          }
+        },
+        "operationId": "getGeometryByGeomId",
+        "parameters": [
+          {
+            "name": "geomId",
+            "in": "path",
+            "description": "Geometry ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "_view",
+            "in": "query",
+            "description": "View. If none selected, defaults to 'geometryview'.",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "geometryview",
+                "centroid",
+                "simplifiedgeom",
+                "alternates"
+              ]
+            }
+          },
+          {
+            "name": "_format",
+            "in": "query",
+            "description": "Media type. If none selected, defaults to 'text/html' or whatever is in the Accept header.",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "enum": [
+                "text/html",
+                "application/json",
+                "text/turtle",
+                "text/plain"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instances that contain specified position",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "searchGeomByPosition",
+        "parameters": [
+          {
+            "name": "x",
+            "in": "path",
+            "description": "x",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": 142.111
+            }
+          },
+          {
+            "name": "y",
+            "in": "path",
+            "description": "y",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": -32.1231
+            }
+          },
+          {
+            "name": "crs",
+            "in": "query",
+            "description": "crs",
+            "required": false,
+            "schema": {
+              "type": "number"
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}/dataset/{dataset_id}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instances that contain specified position for a specified dataset",
```

Comment:
"Get individual geometries that contain the specified position and are associated with the dataset specified"

Reply:
How does this actually work? Is it 
(a) by chasing back any `geox:isGeometryOf` properties (which could be 1:n)
(b) parsing the identifier? (probably only 1:1)

Reply:
I had something to do with the implementation of geox:isGeometryOf. It is generated by parsing the identifiers. This is just because that is convenient and expedient at the moment, in the future we could use more sophisticated mechanisms that map single geometry identifiers to many datasets. One way that might work is that datasets themselves are harvested and a lookup table is built mapping a dataset resource id back to the geometry by following geox:hasGeometry and storing this table.

Reply:
`geox:isGeometryOf owl:inverseOf geo:hasGeometry .`

(different namespaces)

Reply:
@dr-shorthair conceptually, it ought to be option (a).

implementation-wise, it's neither at the moment (actually sort-of (b)). there's a query that goes to the backend and it expects a table with these columns:
<geom-id, dataset, the_actual_geometry>

As @benjaminleighton mentions, it was convenient/expedient to do so. So the `geox:isGeometryOf` and the `geom URI` are both constructed at runtime.

The alternative is what @benjaminleighton suggests, which is to populate a separate table with geom and isGeometryOf values (features).

Reply:
AFAICT it is not necessary to own-up to the actual mechanism, provided the `GetDatasetIDs` operation is available.